From 93446de38d0d2942d6a53593ccc674edb0b886db Mon Sep 17 00:00:00 2001
From: Isaku Yamahata <isaku.yamahata@intel.com>
Date: Thu, 7 Sep 2023 11:20:16 -0700
Subject: [PATCH 213/215] KVM: Add missing trace names for KVM exit reason

Add AP_RESET_HOLD, X86_BUS_LOCK, XEN, RSCV_SBI and RISCV_CSR.

Signed-off-by: Isaku Yamahata <isaku.yamahata@intel.com>
---
 include/trace/events/kvm.h | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/include/trace/events/kvm.h b/include/trace/events/kvm.h
index 3bd31ea23fee..a04951a8c3a2 100644
--- a/include/trace/events/kvm.h
+++ b/include/trace/events/kvm.h
@@ -17,7 +17,9 @@
 	ERSN(NMI), ERSN(INTERNAL_ERROR), ERSN(OSI), ERSN(PAPR_HCALL),	\
 	ERSN(S390_UCONTROL), ERSN(WATCHDOG), ERSN(S390_TSCH), ERSN(EPR),\
 	ERSN(SYSTEM_EVENT), ERSN(S390_STSI), ERSN(IOAPIC_EOI),          \
-	ERSN(HYPERV), ERSN(ARM_NISV), ERSN(X86_RDMSR), ERSN(X86_WRMSR)
+	ERSN(HYPERV), ERSN(ARM_NISV), ERSN(X86_RDMSR), ERSN(X86_WRMSR),	\
+	ERSN(AP_RESET_HOLD), ERSN(X86_BUS_LOCK), ERSN(XEN),		\
+	ERSN(RISCV_SBI), ERSN(RISCV_CSR)
 
 TRACE_EVENT(kvm_userspace_exit,
 	    TP_PROTO(__u32 reason, int errno),
-- 
2.39.2

